[2022-08-06 19:38:56,502] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-13T00:00:00+00:00 [queued]>
[2022-08-06 19:38:56,519] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-13T00:00:00+00:00 [queued]>
[2022-08-06 19:38:56,519] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 19:38:56,519] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 19:38:56,519] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 19:38:56,536] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-05-13 00:00:00+00:00
[2022-08-06 19:38:56,542] {standard_task_runner.py:52} INFO - Started process 2820 to run task
[2022-08-06 19:38:56,546] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-05-13T00:00:00+00:00', '--job-id', '869', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpvd2h7yq7', '--error-file', '/tmp/tmp4h8yeh8m']
[2022-08-06 19:38:56,546] {standard_task_runner.py:80} INFO - Job 869: Subtask second_task
[2022-08-06 19:38:56,629] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-13T00:00:00+00:00 [running]> on host b29c168c1666
[2022-08-06 19:38:56,737] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-05-13T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-13T00:00:00+00:00
[2022-08-06 19:38:56,738] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 19:38:56,738] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 19:38:56,755] {subprocess.py:85} INFO - Output:
[2022-08-06 19:38:56,757] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 19:38:56,758] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 19:38:56,799] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220513T000000, start_date=20220806T193856, end_date=20220806T193856
[2022-08-06 19:38:56,838] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 19:38:56,878] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 21:52:47,070] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-13T00:00:00+00:00 [queued]>
[2022-08-06 21:52:47,087] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-13T00:00:00+00:00 [queued]>
[2022-08-06 21:52:47,087] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:52:47,087] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 21:52:47,087] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:52:47,104] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-05-13 00:00:00+00:00
[2022-08-06 21:52:47,111] {standard_task_runner.py:52} INFO - Started process 3003 to run task
[2022-08-06 21:52:47,114] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-05-13T00:00:00+00:00', '--job-id', '867', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpdspksyo6', '--error-file', '/tmp/tmp2pisnjls']
[2022-08-06 21:52:47,115] {standard_task_runner.py:80} INFO - Job 867: Subtask second_task
[2022-08-06 21:52:47,208] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-13T00:00:00+00:00 [running]> on host c3f731879890
[2022-08-06 21:52:47,308] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-05-13T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-13T00:00:00+00:00
[2022-08-06 21:52:47,310] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 21:52:47,310] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 21:52:47,324] {subprocess.py:85} INFO - Output:
[2022-08-06 21:52:47,326] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 21:52:47,326] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 21:52:47,358] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220513T000000, start_date=20220806T215247, end_date=20220806T215247
[2022-08-06 21:52:47,406] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 21:52:47,438] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 21:58:14,953] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-13T00:00:00+00:00 [queued]>
[2022-08-06 21:58:14,973] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-13T00:00:00+00:00 [queued]>
[2022-08-06 21:58:14,973] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:58:14,973] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 21:58:14,973] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 21:58:14,997] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-05-13 00:00:00+00:00
[2022-08-06 21:58:15,005] {standard_task_runner.py:52} INFO - Started process 2811 to run task
[2022-08-06 21:58:15,008] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-05-13T00:00:00+00:00', '--job-id', '872', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp2acr_oyi', '--error-file', '/tmp/tmpe_f375lx']
[2022-08-06 21:58:15,009] {standard_task_runner.py:80} INFO - Job 872: Subtask second_task
[2022-08-06 21:58:15,121] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-13T00:00:00+00:00 [running]> on host bc7e154a4fdd
[2022-08-06 21:58:15,243] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-05-13T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-13T00:00:00+00:00
[2022-08-06 21:58:15,244] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 21:58:15,245] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 21:58:15,263] {subprocess.py:85} INFO - Output:
[2022-08-06 21:58:15,265] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 21:58:15,265] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 21:58:15,302] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220513T000000, start_date=20220806T215814, end_date=20220806T215815
[2022-08-06 21:58:15,342] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 21:58:15,384] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:13:14,675] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-13T00:00:00+00:00 [queued]>
[2022-08-06 22:13:14,700] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-13T00:00:00+00:00 [queued]>
[2022-08-06 22:13:14,700] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:13:14,700] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:13:14,700] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:13:14,729] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-05-13 00:00:00+00:00
[2022-08-06 22:13:14,738] {standard_task_runner.py:52} INFO - Started process 2816 to run task
[2022-08-06 22:13:14,742] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-05-13T00:00:00+00:00', '--job-id', '867', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp9dit8uqf', '--error-file', '/tmp/tmp3j1enfba']
[2022-08-06 22:13:14,742] {standard_task_runner.py:80} INFO - Job 867: Subtask second_task
[2022-08-06 22:13:14,829] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-13T00:00:00+00:00 [running]> on host 1250c3b659cd
[2022-08-06 22:13:14,931] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-05-13T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-13T00:00:00+00:00
[2022-08-06 22:13:14,932] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:13:14,932] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:13:14,950] {subprocess.py:85} INFO - Output:
[2022-08-06 22:13:14,952] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:13:14,953] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:13:15,013] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220513T000000, start_date=20220806T221314, end_date=20220806T221315
[2022-08-06 22:13:15,077] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:13:15,117] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-08-06 22:24:31,798] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-13T00:00:00+00:00 [queued]>
[2022-08-06 22:24:31,812] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-13T00:00:00+00:00 [queued]>
[2022-08-06 22:24:31,812] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:24:31,812] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-06 22:24:31,812] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-06 22:24:31,828] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): second_task> on 2022-05-13 00:00:00+00:00
[2022-08-06 22:24:31,833] {standard_task_runner.py:52} INFO - Started process 2857 to run task
[2022-08-06 22:24:31,835] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2022-05-13T00:00:00+00:00', '--job-id', '869', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmplxidycqz', '--error-file', '/tmp/tmpoxl1t1w1']
[2022-08-06 22:24:31,836] {standard_task_runner.py:80} INFO - Job 869: Subtask second_task
[2022-08-06 22:24:31,890] {task_command.py:371} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2022-05-13T00:00:00+00:00 [running]> on host b215076695c3
[2022-08-06 22:24:31,962] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2022-05-13T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-13T00:00:00+00:00
[2022-08-06 22:24:31,963] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-06 22:24:31,963] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-08-06 22:24:31,975] {subprocess.py:85} INFO - Output:
[2022-08-06 22:24:31,977] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-08-06 22:24:31,977] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-06 22:24:32,003] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20220513T000000, start_date=20220806T222431, end_date=20220806T222432
[2022-08-06 22:24:32,047] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-06 22:24:32,074] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
